spec/mailers/course_update_email_mailer_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe CourseUpdateEmailMailer, type: :mailer do
+  let(:course) { create(:course, :with_accrediting_provider, updated_at: DateTime.new(2001, 2, 3, 4, 5, 6)) }
+  let(:mail) { described_class.course_update_email(course, "name") }
+
+  before do
+    course
+    mail
+  end
+
+  context "sending an email to a user" do
+    it "sends an email with the correct template" do
+      expect(mail.govuk_notify_template).to eq(Settings.govuk_notify.course_update_email_template_id)
+    end
+
+    it "sends an email to the correct email address" do
+      expect(mail.to).to eq([course.accrediting_provider.email])
```

Comment:
I'm not sure if that is the correct email address to use? Should it not be the users that are associated with that provider?

Reply:
It's not clear from the ticket who the email is meant to be sent to - it just says "As an accredited body, I need to receive a notification". I think there's a bit of policy decision here to be made! I'll bring it up at standup

Reply:
I did what Dan suggested and adding a user argument, which we can then draw the correct email from. Don't need to worry about it for now!